### DISCUSSION TITLE
EVSE Set Configuration and EVSE Set DC Charging Parameters

### DISCUSSION THREAD

**Discussion by amlac** [Sevenstax/FreeV2G#136] [Q&A]
Hi,

We are trying to send the commands EVSE Set Configuration and EVSE Set DC Charging Parameters but we are always receiving malformed data response. 

We are trying to send the examples at White Beet User Manual:

**EVSE Set Configuration:**
C0 27 60 00 00 2A       (ReqID: 0; Payload: 42 byte)
0F 2b 34 39 2a 31 32 33 2a 34 35 36 2a 37 38 39
                                    (EVSE ID DIN: +49*123*456*789)
0F 44 45 2a 41 32 33 2a 45 34 35 42 2a 37 38 43
                                   (EVSE ID ISO: DE*A23*E45B*78C)
02 00 01                     (Protocols DIN70121 and ISO15118 supported)
02 00 01                     (Payment method EIM and PNC)
01 01                          (Energy transfer mode DC extended)
00                               (Certificate Installation Supported)
00                               (Certificate Update Supported)
00 C1

**EVSE Set DC Charging Parameters:**
C0 27 62 02 00 18                   (ReqID: 2; Payload: 24 byte)
00                                            (Isolation Level invalid)
00 00 00                                  (Min. Voltage 0V)
00 00 00                                  (Min. Current 0V)
00 04 02                                  (Max. Voltage 400V)
00 05 01                                  (Max. Current 50A)
00 14 03                                  (Max. Power 20kW)
01 00 02 00                             (Current Regulation Tolerance 2A)
00 05 81                                  (Peak Current Ripple 0.5A)
00                                            (Status: Ready)
00 C1

Are the examples in the manual correct or what could be wrong?

Thanks.

**Comment by Volodymyr233** [comment]
Hi! Were you able to overcome this problem (EVSE Set Configuration and EVSE Set DC Charging Parameters)? I also ran into her. Thanks.
